pootle/apps/pootle_format/formats/mozilla_lang.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from pootle_store.constants import FUZZY, UNTRANSLATED
+from pootle_store.diff import DiffableStore
+from pootle_store.syncer import StoreSyncer, UnitSyncer
+
+
+class LangUnitSyncer(UnitSyncer):
+
+    @property
+    def target(self):
+        if self.isfuzzy:
+            return ""
+        return self.unit.source
+
+
+class LangStoreSyncer(StoreSyncer):
+    unit_sync_class = LangUnitSyncer
+
+
+class DiffableLangStore(DiffableStore):
+
+    def get_unit_state(self, file_unit):
```

Comment:
Its not really a DiffableStore method - more of a DiffableUnit one - but in this case it needs to happen here as we have access to both stores